gauntlet/packages/gauntlet-solana/src/commands/internal/solana.ts

```diff
@@ -107,7 +143,41 @@ export default abstract class SolanaCommand extends WriteCommand<TransactionResp
     }
     const signedTx = await this.wallet.signTransaction(tx)
     logger.loading('Sending tx...')
-    return await sendAndConfirmRawTransaction(this.provider.connection, signedTx.serialize())
+    try {
+      return await sendAndConfirmRawTransaction(this.provider.connection, signedTx.serialize())
+    } catch (error) {
+      // Retry mechanism with greater priority fees
+      if (
+        error instanceof SendTransactionError &&
+        error.message.includes('congestion') &&
+        validateRetryPriorityInput(this.flags.priorityRetry)
+      ) {
+        const [percentBump, numberOfRetries] = this.flags.priorityRetry.split(',')
```

Comment:
after validation succeeds, you can print out the new priority fee and how much it was bumped by

ex: "Retry with higher priority fee (x% bump): Y"